tom_hermes/hermes.py

```diff
@@ -24,59 +25,52 @@
 logger = logging.getLogger(__name__)
 #logger.setLevel(logging.DEBUG)
 
-HERMES_URL = os.getenv('HERMES_BASE_URL', 'http://hermes.lco.global')
+HERMES_URL = settings.HERMES_API_URL if hasattr(settings, 'HERMES_API_URL') else os.getenv('HERMES_BASE_URL', 'http://hermes.lco.global')
```

Comment:
I've set this up to check your settings first for `HERMES_API_URL`, which you will set if you are using tom_nonlocalizedevents most likely... Otherwise it can fall back on the env variable you were checking before I guess.